dom-manipulation/main.js

```diff
@@ -0,0 +1,22 @@
+var hotButton = document.querySelector('.hot-button');
+var clickCount = document.querySelector('.click-count');
+var counter = 1;
+
+hotButton.addEventListener('click', counterIncrement);
+
+function counterIncrement(event) {
+  clickCount.textContent = 'Clicks: ' + counter++;
```

Comment:
In this form, your first click won't update the click count. Remember how the `++` operator works as related to the variable it is acting on!